os/board/rtl8721csm/src/component/common/bluetooth/realtek/sdk/example/ble_central/ble_tizenrt_client.c

```diff
@@ -88,7 +88,7 @@ trble_result_e rtw_ble_client_scan_whitelist_add(trble_addr *addr)
     {
         if(false == os_mutex_create(&ble_tizenrt_modify_whitelist_sem))
         {
-            printf("\create sem fail! \n");
+            printf("create sem fail! \n");
```

Comment:
all other logs are `debug_printf`, but only this is `printf`.
Could you check that we can use  `debug_printf` in here also?

Reply:
fix done, please check via https://github.com/Samsung/TizenRT/pull/5637/commits/fcafbac57359fd9cdabc0dd9a1f2e0d97c931cae